app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -127,6 +130,27 @@ class MembersFragment : BaseFragment() {
                                 isRecyclerView = true
                             }
                         }
+                        else
+                        {
+                            if(filterMap.containsKey("location")&&filterMap["location"] != "")
+                            {
+                                location_check = false
+                                val query = filterMap["location"].toString()
+                                for (user in membersViewModel.userList) {
+                                    // ""+ to convert String to CharSequence
+                                    if (("" + user.location).contains(query, ignoreCase = true)) {
+                                        location_check = true
+                                    }
+                                }
```

Comment:
use `userList.any{}` instead

Reply:
On it, Thanks for the suggestion :)

Reply:
Also why do you need to convert it to char sequence is it not a string?

Reply:
I just copied the loop from searchuser() function, and modified it a little bit. And yes you are right, there is no need to convert string into char sequence.

Reply:
@epicadk while using,
membersViewModel.userList.any {it.location == filterMap["location"].toString()}

It always return false, Why so?